jina/clients/base/websocket.py

```diff
@@ -64,6 +64,15 @@ def _response_handler(response):
                         if response.header.request_id in request_buffer:
                             future = request_buffer.pop(response.header.request_id)
                             future.set_result(response)
+                            if (
+                                response.header.status.exception.name
+                                == "<class 'jina.excepts.InternalNetworkError'>"
```

Comment:
We should find a better way to handle this.

Reply:
Agree, but couldn't come up with one

Reply:
found a way ✨